spitfire/runtime/udn_test.py

```diff
@@ -63,6 +66,13 @@ def test_in_search_list_object(self):
         self.assertIn('boom', template.placeholder_cache)
         self.assertEqual(udn.resolve_placeholder('boom', template, None), 'bam')
 
+    def test_method_cycle_elimination(self):
```

Comment:
Yeah, I just updated the test to check the type of the entry in the placeholder cache, which should fulfill this.